modules/client-configuration/pages/contact-methods-saltbundle.adoc

```diff
@@ -21,6 +21,18 @@ If you bootstrap new clients, registration with the Salt Bundle is the default m
 You can switch existing clients to the Salt Bundle method.
 If you switch, the [package]``salt-minion`` package and its dependencies will stay installed.
 
+=== Using the Salt Bundle with the Salt Minion
+
+The Salt Bundle can be used with the Salt Minion managed by the Salt Master other than {productname} Server at the same time.
+If the Salt Bundle is installed on a client {productname} Server will manage its configuration files, the configuration files of [literal]``salt-minion`` will not be managed in this case.
+For more information, see xref:client-configuration:registration-cli.adoc#_salt_bundle_configuration[Salt Bundle configuration].
```

Comment:
```suggestion
The Salt Bundle can be used as a client registration method even it the Salt Master runs on an own host, separately from the {productname} Server.
If the Salt Bundle is installed on a client {productname} Server will manage the client's configuration files, but the configuration files of [literal]``salt-minion`` will not be managed in this case.
For more information, see xref:client-configuration:registration-cli.adoc#_salt_bundle_configuration[Salt Bundle configuration].
```

Reply:
The Salt Bundle is not the registration method by itself. In your suggestion it's loosing the exact meaning of the. The second sentance not about all the client's configuration files, but exactly about Salt Bundle or Salt Minion config files.

Reply:
ok, I see.  In this case we could consider to repeat the term "the Salt Bundle" or "the Client's Salt Bundle" instead of "its".  But this isn't that important.

Reply:
Fixed slightly different way with https://github.com/uyuni-project/uyuni-docs/pull/1480/commits/c350233476de0eb428eb32af2213a31797545398, is it fine now?

Reply:
thanks, sounds good now!